examples/paint/README.md

```diff
@@ -0,0 +1 @@
+A regular paint, the ability to draw free-form lines using Shape Painter, the example implements color change using RGB, reading the pixel color using the Read Pixels extension.
```

Comment:
`A drawing app in which you can draw free form lines.`

And the long description:
` The example uses the shape painter to draw on the screen. And implements a color picker using the Read Pixels extension`